addons/website/static/tests/tours/snippet_table_of_content.js

```diff
@@ -42,6 +42,7 @@ wTourUtils.registerWebsitePreviewTour('snippet_table_of_content', {
     {
         content: "Click on the first TOC's title",
         trigger: 'iframe .s_table_of_content:eq(0) h1',
+        extra_trigger: "div:not(.o_loading_dummy) > #oe_snippets button[data-action='save']:not([disabled])",
```

Comment:
why not the same as everywhere else when entering edit mode?
```suggestion
        extra_trigger: " '#oe_snippets.o_loaded',",
```

And it should be part of the `clickOnEdit` definition, that helper should wait for the edit mode to start in a next step. It will prevent such race condition.
But in earlier version where it was introduced, I'll create a task for that, just adapt this particular occurence in this PR.